nym-vpn-core/Makefile

```diff
@@ -6,6 +6,27 @@ include platform.mk
 export MACOSX_DEPLOYMENT_TARGET = 10.13
 export IPHONEOS_DEPLOYMENT_TARGET = 16.0
 
+# Project root replaced with /buildroot in reproducible builds
+PROJECT_ROOT=$(shell dirname $(CURDIR))
+
+# Rust compiler sysroot that typically points to ~/.rustup/toolchains/<toolchain>
+RUST_COMPILER_SYS_ROOT=$(shell rustc --print sysroot)
+
+# Rust flags used in reproducible builds to replace common paths with defaults
+IDEMPOTENT_RUSTFLAGS=-C link-args=-Wl,--build-id=none --remap-path-prefix $(HOME)= --remap-path-prefix $(PROJECT_ROOT)= --remap-path-prefix $(RUST_COMPILER_SYS_ROOT)=
```

Comment:
`--build-id=none` does no work on macos so we need to condition it to linux only.